scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/job/JobResultImpl.java

```diff
@@ -57,6 +59,9 @@
     /** List of precious results */
     private Map<String, TaskResult> preciousResults = null;
 
+    /** Map of jobResults which merged in the end of the job */
+    private Map<String, Serializable> JobMap = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "JobMap" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)